src/Apps/Legion/Models/Wizard.types.ts

```diff
@@ -44,6 +44,12 @@ export interface IViewEntity extends IBaseEntity {
     /** the resolved view model of the `Type` the entity has */
     type: IViewType;
 }
+
+export interface IEntityCounters {
+    new: number;
```

Comment:
`new` feels like it's going to give use sadness at some point (like `delete`). Could we use "created" or something similar?

Reply:
@msnyder-msft left a comment asking for `new` it is consistent with schema stuff. Should I revert to `discovered`?

Reply:
ahhh that could be fair. Fine. Revert it if you think that's a problem. I think acessing liek like `myObject.new` **should** be fine, but whatevs. Don't feel strongly.

Reply:
I would rather just send it @montgomp, thoughts?